region/brinstar/red.json

```diff
@@ -3070,6 +3217,33 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Leave With Runway (Power Bomb Blocks Intact)",
+                  "notable": false,
+                  "requires": [
+                    {"obstaclesCleared": ["A"]},
+                    {"obstaclesNotCleared": ["B"]}
+                  ],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 29,
+                      "openEnd": 0
+                    }
+                  },
+                  "note": [
+                    "This runway requires the PB blocks to be intact, but the sidehoppers to be dead.",
+                    "This is relevant because it can limit the player's options for killing the Sidehoppers."
+                  ],
+                  "devNote": [
+                    "Killing the Sidehoppers requires going to 3 and back.",
+                    "FIXME: A shorter runway could be added assuming a PB was laid as far left as possible."
```

Comment:
placing a PB against the left wall kills the Sidehoppers, or if some survive, they can safely be killed with pea-shooter. The remaining runway is 12 tiles + 1 open end